Makefile

```diff
@@ -6,7 +6,7 @@ BUILD_DATE?=$(shell date +%Y-%m-%dT%H:%M:%S%z)
 LOCAL_PACKAGES?=app app/controllers app/options app/watchers utils
 IMG_NAMESPACE?=cloudnativelabs
 GIT_COMMIT=$(shell git describe --tags --dirty)
-GIT_BRANCH?=$(shell git rev-parse --abbrev-ref HEAD)
+GIT_BRANCH?=$(shell git rev-parse --abbrev-ref HEAD | tr -d '/')
```

Comment:
another option would be to use Makefile substitution to filter `/` characters

```
$(subst from,to,text)
```

This might be better since someone might set GIT_BRANCH externally.

Reply:
Updated per recommendation. Also made it specific to the building the docker image which is the only part that has issues with slashes in case someone needed to use that to refer to the actual git branch.